README.md

```diff
@@ -47,16 +47,19 @@ export AZURE_STORAGE_ACCESS_KEY="<youraccountkey>"
 ```
 ### Windows
 
-```cmd
-setx AZURE_STORAGE_ACCOUNT "<youracountname>"
-setx AZURE_STORAGE_ACCESS_KEY "<youraccountkey>"
+```powershell
+$env:AZURE_STORAGE_ACCOUNT="<youracountname>"
```

Comment:
Since the sample is using AAD for authentication, you can remove these env vars.